apps/web/src/server/views/applications/create/_region.njk

```diff
@@ -9,7 +9,7 @@
 {% block stepForm %}
 	{{ govukCheckboxes({
 		idPrefix: "selectedRegionsNames",
-		name: "selectedRegionsNames",
+		name: "selectedRegionsNames[]",
```

Comment:
⚠️ this fixes a bug on the regions commit. 
Hopefully these changes will be released at the same time and nobody will notice the bug